sci-libs/torchvision/files/torchvision-include-dir.patch

```diff
@@ -0,0 +1,15 @@
+diff -Naur a/setup.py b/setup.py
```

Comment:
Put descriptions on top of the patch files, documenting the purpose of this patch and whether it is submitted upstream.